Presentations/03-data-wrangling.Rmd

```diff
@@ -98,17 +98,17 @@ name: intro
 .panelset[
 
 .panel[.panel-name[If You Attended Session 2]
-1. Go to the `dime-r-training-mar2023` folder that you created yesterday, and open the `dime-r-training-mar2023` R project that you created there.
+1. Go to the `dime-r-training-mar2024` folder that you created yesterday, and open the `dime-r-training-mar2024` R project that you created there.
 ]
 
 .panel[.panel-name[If You Did Not Attend Session 2]
-1. Create a folder named `dime-r-training-mar2023` in your preferred location in your computer.
+1. Create a folder named `dime-r-training-mar2024` in your preferred location in your computer.
```

Comment:
this is the tab that should have the instructions with `use_course()`

Reply:
It is added now